rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -53,7 +51,7 @@
     },
     "call_number": {
       "title": "Call number",
-      "description": "Call number of the item.",
+      "description": "Call number of the item. This field is optional",
```

Comment:
same here

Reply:
Yes, same as before, I recommend deleting the description.